### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by PromanSEW**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -50,24 +50,24 @@
     <!-- Dependencies -->
     <android.version>4.1.1.4</android.version>
     <okhttp.version>3.9.1</okhttp.version>
-    <animal.sniffer.version>1.14</animal.sniffer.version>
+    <animal.sniffer.version>1.16</animal.sniffer.version>
 
     <!-- Adapter Dependencies -->
-    <rxjava.version>1.3.0</rxjava.version>
+    <rxjava.version>1.3.4</rxjava.version>
     <rxjava2.version>2.0.0</rxjava2.version>
```
Comment (PromanSEW) [inline pom.xml]: https://github.com/ReactiveX/RxJava/releases
What version should be used?

File: pom.xml [hunk 2]
```diff
@@ -50,24 +50,24 @@
...
     <guava.version>19.0</guava.version>
-    <scala.version>2.12.3</scala.version>
+    <scala.version>2.12.4</scala.version>
 
     <!-- Converter Dependencies -->
     <gson.version>2.8.2</gson.version>
     <protobuf.version>3.0.0</protobuf.version>
```
Comment (PromanSEW) [inline pom.xml]: https://github.com/google/protobuf/releases
What version should be used?

**Comment by PromanSEW** [comment]
@JakeWharton @swankjesse please review

**Comment by jaredsburrows** [comment]
@PromanSEW You might want to update each dependency in a separate commit.

**Pull request closed without merging**